File: BEST-App-main/lib/model/payment.dart
class Payment {
  final int id;
  final String name;
  final String description;

  Payment(this.id, this.name, this.description);

  static Map<String, dynamic> toJson(Payment payment) {
    return {
      'id': payment.id,
      'name': payment.name,
      'description': payment.description
    };
  }
}

File: spring-best/src/main/java/com/best/springbest/model/Payment.java
package com.best.springbest.model;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import javax.persistence.*;

@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "payments")
public class Payment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private String name;

    private String description;
}
